### DISCUSSION TITLE
Please add descriptions to the API docs in v3

### DISCUSSION THREAD

**Discussion by ninjasort** [vuetifyjs/vuetify#16096] [General]
I'm seeing a lot of missing descriptions in the v3 docs. Can you please fix that or redirect to v2 so we know what is fact vs fiction?

**Comment by dhait** [comment]
Yes, I must say this is incredibly annoying when trying to use components, such as slots for example.

Example: [https://github.com/vuetifyjs/vuetify/blob/next/packages/api-generator/src/locale/en/v-list-item.json](https://github.com/vuetifyjs/vuetify/blob/next/packages/api-generator/src/locale/en/v-list-item.json)
